<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgBackgroundProperties</class>
 <widget class="QDialog" name="DlgBackgroundProperties">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>508</width>
    <height>367</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>300</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Sans Serif</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Background properties</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>8</number>
   </property>
   <property name="topMargin">
    <number>8</number>
   </property>
   <property name="rightMargin">
    <number>8</number>
   </property>
   <property name="bottomMargin">
    <number>8</number>
   </property>
   <item>
    <widget class="QRadioButton" name="SameBackgroundRD">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>If this option is selected, the slide will have no background definition  but will use the same background as the previous slide:
- If the previous slide does not have a background definition either, the check continues backwards until a slide is found which does have a background definition.
- If no previous slide has a background definition, then a black frame will be used.</string>
     </property>
     <property name="text">
      <string>Same background as preceding object</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="NewBackgroundRD">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>If this option is selected, the slide will have its own background definition.</string>
     </property>
     <property name="text">
      <string>Select a new background</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="Preview">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="text">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="scrollArea" native="true">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="1" column="0">
          <widget class="QLabel" name="FirstColorLabel">
           <property name="text">
            <string notr="true"/>
           </property>
           <property name="pixmap">
            <pixmap>:/img/color.png</pixmap>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="BrushTypeLabel">
           <property name="text">
            <string notr="true"/>
           </property>
           <property name="pixmap">
            <pixmap>:/img/color_fill.png</pixmap>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="cBackgroundComboBox" name="BackgroundCombo">
             <property name="minimumSize">
              <size>
               <width>140</width>
               <height>68</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>140</width>
               <height>68</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Allows you to select an image.</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="BackgroundComboSpacer">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string notr="true"/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="ImageFileLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="text">
            <string notr="true"/>
           </property>
           <property name="pixmap">
            <pixmap>:/img/object_image.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="BrushTypeCombo">
           <property name="toolTip">
            <string>Allows you to select a brush type in the following list:
- Solid brush: Allow to selects a plain color.
- Gradient 2 colors: Allow to select a gradient from 2 colors
- Gradient 3 colors: Allow to select a gradient from 3 colors
- Image from library: Allow to select a predefined background
- Image disk: Allow to select a photo or an image from a file on the disk.</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="ImageLibraryLabel">
             <property name="minimumSize">
              <size>
               <width>24</width>
               <height>24</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>24</width>
               <height>24</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap>:/img/object_image.png</pixmap>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="ImageLibraryLabel2">
             <property name="text">
              <string notr="true"/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="OrientationLabel">
           <property name="text">
            <string notr="true"/>
           </property>
           <property name="pixmap">
            <pixmap>:/img/Distance.png</pixmap>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <property name="spacing">
            <number>2</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QRadioButton" name="KeepRatioRB">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="toolTip">
              <string>If this box is checked, the image isn't deformed and you can open the Image framing dialog to select a part of the image.</string>
             </property>
             <property name="text">
              <string>Keep aspect ratio</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="ImageEditCorrectBT">
             <property name="toolTip">
              <string>Opens the Image framing dialog to select a part of the image.</string>
             </property>
             <property name="text">
              <string notr="true"/>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>:/img/EditImage.png</normaloff>:/img/EditImage.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="cCGrdOrientationComboBox" name="OrientationCombo">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>48</width>
               <height>24</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>48</width>
               <height>24</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Allows you to select the gradient orientation.</string>
             </property>
             <property name="iconSize">
              <size>
               <width>64</width>
               <height>16</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="IntermPosSlider">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>65</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>65</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Allows you to define the position (in percentage) of the 3rd gradient color.</string>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="value">
              <number>10</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="IntermPosED">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="toolTip">
              <string>Allows you to define the position (in percentage) of the 3rd gradient color.</string>
             </property>
             <property name="suffix">
              <string notr="true">%</string>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="value">
              <number>10</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="OrientationSpacer">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string notr="true"/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="cCColorComboBox" name="FirstColorCombo">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>48</width>
               <height>24</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>48</width>
               <height>24</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Allows you to select the first color.</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="cCColorComboBox" name="IntermColorCombo">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>48</width>
               <height>24</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>48</width>
               <height>24</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Allows you to select the second color.</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="cCColorComboBox" name="FinalColorCombo">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>48</width>
               <height>24</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>48</width>
               <height>24</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Allows you to select the last color.</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="FirstColorSpacer">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string notr="true"/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="6" column="1">
          <widget class="QRadioButton" name="FullFillRB">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>If this box is checked, the image fills the screen, even if it has to be deformed.</string>
           </property>
           <property name="text">
            <string>Full filling (deform image)</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLineEdit" name="ImageFileED">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string notr="true"/>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="ImageFileBT">
             <property name="toolTip">
              <string>Allows you to select a photo or an image from a file on the disk.</string>
             </property>
             <property name="text">
              <string notr="true">...</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>:/img/FolderOpen.png</normaloff>:/img/FolderOpen.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>4</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="OKBT">
       <property name="toolTip">
        <string>Accept modification and exit dialog box</string>
       </property>
       <property name="text">
        <string>&amp;Ok</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/img/action_ok.png</normaloff>:/img/action_ok.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="UndoBT">
       <property name="toolTip">
        <string>Undo last action (CTRL+Z)</string>
       </property>
       <property name="text">
        <string>Undo</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/img/undo24.png</normaloff>:/img/undo24.png</iconset>
       </property>
       <property name="shortcut">
        <string>Ctrl+Z</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="CancelBt">
       <property name="toolTip">
        <string>Cancel modification and exit dialog box</string>
       </property>
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/img/action_cancel.png</normaloff>:/img/action_cancel.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="HelpBt">
       <property name="toolTip">
        <string>Show help about this dialogue box</string>
       </property>
       <property name="text">
        <string>&amp;Help</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/img/help.png</normaloff>:/img/help.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>cCColorComboBox</class>
   <extends>QComboBox</extends>
   <header>../ffDiaporama_lib/cColorComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>cCGrdOrientationComboBox</class>
   <extends>QComboBox</extends>
   <header>../ffDiaporama_lib/cGrdOrientationComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>cBackgroundComboBox</class>
   <extends>QComboBox</extends>
   <header>../ffDiaporama_lib/cBackgroundComboBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
